### ISSUE TITLE
feature-request/on-end-reached

### ISSUE THREAD

**Issue by dougg0k** [marcocesarato/react-native-big-list#4]
I would like to request a feature if that is possible, to easily be able to paginate on scroll. Like the feature in Flatlist https://reactnative.dev/docs/flatlist#onendreached

**Comment by marcocesarato** [comment]
Hi @dougg0k, thanks for the suggestion, it is available on release `1.2.4`.

Let me know if everything works fine.

**Comment by dougg0k** [comment]
Hi @marcocesarato, It's working great, thanks! 

I noticed in your last commit that you added onRefresh, but doesn't seem to be available. Maybe you didnt added to the package yet.

**Comment by marcocesarato** [comment]
Yes, it will be available on next release after some test and isn't available yet.
